Blog_part_2.md

```diff
@@ -0,0 +1,324 @@
+
+Now let's take a look at the type of machines we can run our container nodes on. Run the following command to get a list:
+
+    $ gcloud compute machine-types list
+
+You can also visit the [Google Compute Machine Type Documenation](https://cloud.google.com/compute/docs/machine-types) to learn more.
+
+In our case we're going to use some higher cpu machines as CPU is often the limiting factor when running locust. The command below uses the `n1-highcpu-8` machine type. This is a High-CPU machine type with 8 virtual CPUs and 7.20 GB of memory. Obviously the more powerful machines you use the more expensive they are. Please use what is comfortable for you. Just note that if you notice the locust system hitting failures you may need to either increase the CPU or Memory of your machines or add more machines.
+
+Since we have 7 workers nodes with 1 controller we need to let container engine know by passing in the `--num-nodes` parameter with a value of 8 in our case. The command looks like so:
+
+    $ gcloud container clusters create CLUSTER-NAME --machine-type=n1-highcpu-8 --num-nodes=8
+
+Let's call our cluster here `locust-cluster` so the exact command we will run is:
+
+    $ gcloud container clusters create locust-cluster --machine-type=n1-highcpu-8 --num-nodes=8
+
+Now let's make the cluster our default cluster in this project by adding it to our gcloud config with the following comand:
+
+    $ gcloud config set container/cluster locust-cluster
+
+You can run `gcloud config list` again to confirm the following entry:
+
+    [container]
+    cluster = locust-cluster
+
+Get credential for your cluster.
+
+    $ gcloud container clusters get-credentials locust-cluster
+
+Now let's look at our clusters by issuing the following command:
+
+    $ gcloud container clusters list
+
+Now you should see your locust-cluster. And if you created the example-cluster earlier it should also be listed.
+
+After a few minutes, you'll have a working Kubernetes cluster with three nodes (not counting the Kubernetes master). This gives us our GCP VMs that will be the hosts for our kubernetes pods that we'll deploy next.
+
+Now we're going to get ready to deploy our nodes. This is where we start to interact with Kubernetes more directly via [kubectl](https://kubernetes.io/docs/user-guide/kubectl-overview/). We will be using some simple kubectl commands in this guide. I highly recommend you dig through the documentation to see what all is available. There are many commands for updating or inspecting services. Here is a [kubectl cheat sheet](https://kubernetes.io/docs/user-guide/kubectl-cheatsheet/) for some of the most common commands. 
+
+Now let's deploy some pods. First we need to ensure our `kubectl` command is pointing at our cluster. If you run the following command you should see a list of contexts: 
+
+    $ kubectl config get-clusters
```

Comment:
Do we need to run https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials before this?